### PR TITLE
Change to navigation for SEO scores.

### LINKED ISSUE #146: Views: Remove the /home in the Navigation Logo URL and Make the Link Relative
The top navigation link is currently

``<a class="navbar-brand" href="https://hackgreenville.com/home">``

It should just be `<a class="navbar-brand" href="/">` to avoid SEO issues with duplicate URLs.

Also, there is no need to render the https:// or the domain.  A "/" will do the job here.

@MarkMcDaniels you willing to refresh your GitHub skills with a simple pull request?  The [CONTRIBUTING.md has a section on how to do the fork, clone, branches, PRs and such](https://github.com/hackgvl/hackgreenville-com/blob/develop/CONTRIBUTING.md#contributing-code-to-the-project) if anybody needs a refresher.

### PR DESCRIPTION
**Pull request by MarkMcDaniels**
Closes #146

### CODE REVIEW

**Comment by allella** [comment]
Thanks Mark. I see from your PR that this issues is due to route('home') generated path.

It's causing usability and SEO issues to have both / and /home with the same content and not including.  We could add a rel=canonical meta tag to tell the search engines that /home is the same as /, but we'd still have the issue of people bookmarking or sharing the /home link and then links breaking when /home ceases to exist due to future changes.

@zach2825 there is there a reason we have the /home route doing the same thing as the / route?

image

**Comment by MarkMcDaniels** [comment]
I didn't want to remove the route because I wasn't sure if it was supposed to be removed as well.

**Comment by zach2825** [comment]
> ecause I wasn't sure if it was su

No,  I write the site super quick, intending to go back and do a bunch of cleanup. if you don't mind moving the  `->name('home')` to the / route that would be perfect

**Comment by allella** [comment]
@MarkMcDaniels I guess we're good remove the two routes and just have one like

`Route::get('/', 'HomeController@index')->name('home');`

I can setup a redirect of /home back to / on the live server, to straighten out the search engines if that's deemed necessary, so we don't need to worry about that for now.

**Comment by allella** [comment]
@MarkMcDaniels if you want to update the routes as in the last comment I posted, then I think the resources/views/layouts/top-nav.blade.php change could be reverted back since it should stop resolving that to /home.

**Comment by MarkMcDaniels** [comment]
Gotcha

File: resources/views/layouts/top-nav.blade.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <nav class="navbar navbar-expand-md navbar-dark bg-primary">
     <div class="container-fluid">
-        <a class="navbar-brand" href="{{ route('home') }}">
```
[the remark this answers is not in the document]
Comment (allella) [reply resources/views/layouts/top-nav.blade.php:3]: @MarkMcDaniels if you revert and push a commit for this one, then we should be good to merge this in.  Grazie.

**Comment by allella** [comment]
@MarkMcDaniels you can revert the README.md too as it's causing a merge conflict and I did something similar to your change.

The route file change is the main thing we want to commit on this one now due to all the other changes and things we figured out since starting this issue and PR.

File: resources/views/layouts/top-nav.blade.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+        <a class="navbar-brand" href="{{ rout('home') }}">
```
Comment (allella) [inline resources/views/layouts/top-nav.blade.php:3]: @MarkMcDaniels I think this may be missing an 'e'. Is it rout() or route()?

Comment (MarkMcDaniels) [reply resources/views/layouts/top-nav.blade.php]: wow... Sorry about that.

**Review by bogdankharchenko — APPROVED** [review]

**Review by allella — APPROVED** [review]

**Pull request merged**